janusgraph-driver/src/test/java/org/janusgraph/graphdb/tinkerpop/JanusGraphIoRegistryTest.java

```diff
@@ -115,6 +115,58 @@ public void testJanusGraphPredicatesAsGryo() throws SerializationException {
 
     }
 
+    @Test
+    public void testOldFormatJanusGraphPredicatesAsGryo() throws SerializationException {
+        Graph graph = EmptyGraph.instance();
+        GraphTraversalSource g = graph.traversal();
+
+        // Janus Graph Geo, Text Predicates
```

Comment:
(nitpick) The comment doesn't make much sense as there are no Geo predicates here and only one text predicate.